Containerfile.dataplane

```diff
@@ -7,8 +7,13 @@ RUN apt-get install --yes \
     build-essential \
     protobuf-compiler \
     pkg-config \
-    llvm-16 \
-    musl-tools
+    musl-tools \
+    wget
+
+RUN wget -O /tmp/llvm.sh https://apt.llvm.org/llvm.sh
```

Comment:
Switching from pulling llvm from the Debian repositories to a `curl | sudo bash` kinda situation isn't ideal, as it adds some fragility to our build system. Additionally this will always just pull the latest release, yes? I take it something in our Aya dependencies changed to require the newer LLVM, right? Is there some other image base we could use that would include a compatible version of LLVM in the package repositories?

Reply:
Is llvm-18 not packaged on ubuntu slim-bookworm?

Reply:
Looks like you can specify the version number with the shell script:
> For convenience there is an automatic installation script available that installs LLVM for you.
To install the latest stable version:
>
> ```sh
> bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
> ```
>
> To install a specific version of LLVM:
> 
> ```sh
> wget https://apt.llvm.org/llvm.sh
> chmod +x llvm.sh
> sudo ./llvm.sh <version number>
> ```
Looks there's an [official debian release](https://apt.llvm.org/) for 18  (under qualification branch): `apt-get install llvm-18`

Is there a reason we can't use that @aryan9600?

Reply:
i looked around (not exhaustingly) and i coulnd't find something. i'll take another look and report back.

Reply:
@aryan9600 heya :wave: just checking in?

Reply:
ping